index.js

```diff
@@ -337,6 +338,10 @@ const errorHandler = (res, err, logger) => {
     res.statusCode = 500
     res.end('Internal Server Error')
   }
+
+  if (res.statusCode >= 500) {
```

Comment:
We don't want to record 4xx errors (e.g. "OUTDATED CLIENT" or when the client submits a malformed JSON payload)